db/seeds/questionnaires/mentor_post_assessment.rb

```diff
@@ -3,185 +3,42 @@
-                       options: ['Nee'],
-                       otherwise_label: 'Ja, namelijk om:'
-                     }, {
-                       id: :v25,
-                       type: :radio,
-                       title: 'Je kreeg nu elke donderdag een vragenlijst. Zou je deze liever op een andere dag krijgen?',
-                       options: ['Nee', 'Ja, op maandag', 'Ja, op dinsdag', 'Ja, op woensdag', 'Ja, op vrijdag'],
-                       section_end: true
-                     }, {
-                       section_start: 'Missen van vragenlijsten',
-                       id: :v26,
-                       type: :checkbox,
-                       title: 'Wat waren de redenen dat je wel eens een vragenlijst hebt gemist? (meerdere antwoorden mogelijk)',
-                       options: ['Ik heb nooit een vragenlijst gemist',
-                                 'Ik kreeg geen sms',
-                                 'De link naar de vragenlijst werkte niet',
-                                 'Ik had geen tijd',
-                                 'Ik had geen zin',
-                                 'Ik was het vergeten',
-                                 'Mijn batterij was leeg',
-                                 'Ik zat op dat moment niet met mijn telefoon op wifi',
-                                 'De databundel van mijn telefoon was op',
-                                 'De vragenlijst was al verlopen'
-                       ],
-                       section_end: true
-                     }, {
-                       section_start: 'Volhouden',
-                       id: :v27,
-                       type: :radio,
-                       title: 'Je hebt nu drie weken meegedaan aan dit onderzoek. Denk je dat je ditzelfde onderzoek ook voor zeven maanden zou volhouden?',
-                       options: ['Ja'],
-                       otherwise_label: 'Nee, omdat:',
-                       section_end: true
-                     }, {
-                       section_start: 'Tot slot',
-                       id: :v28,
-                       type: :textarea,
-                       title: 'Wat zou jij willen verbeteren aan de webapp die je de afgelopen drie weken hebt gebruikt?',
-                       section_end: true
-                     }]
+  id: :v1,
+  type: :textarea,
+  title: 'Kan je in één woord beschrijven hoe jij het ervaart jij om jongeren te begeleiden binnen {{je_begeleidingsinitiatief}}?'
+}, {
+  id: :v2,
+  type: :range,
+  title: 'In hoeverre vind jij dat {{deze_student}} vooruitgang heeft geboekt in {{zijn_haar_student}} begeleidingstraject?',
+  foreach: :student,
+  labels: ['Helemaal niet ', 'heel veel']
+}, {
+  id: :v3,
+  type: :range,
+  title: 'Hielp het invullen van de app je om jouw jongeren beter te begeleiden? ',
+  labels: ['Helemaal niet ', 'helemaal wel']
+}, {
+  id: :v3,
+  type: :range,
+  title: 'Hoe moeilijk was het om zo lang elke week de app in te vullen?',
```

Comment:
Neutraler: Hoe ervoer/ervaarde jij het om 8 maanden lang de app in te vullen?

Je bent nu aan het vissen naar een negatieve reactie door woorden zoals "moeilijk"  en "zo lang" te gebruiken.